site/app/models/Gradeable.php

```diff
@@ -267,6 +267,13 @@ public function __construct(Core $core, $details, User $user = null) {
                             break;
                         }
                     }
+                    
+                    $component_details['gcd_grader_id'] = explode(',', trim($details['array_gcd_grader_id'], "{}"))[$i];
+                    $component_details['gcd_grade_time'] = explode(',', trim($details['array_gcd_grade_time'], "{}"))[$i];
```

Comment:
use [DatabaseUtils::fromPGToPHPArray](https://github.com/Submitty/Submitty/blob/master/site/app/libraries/DatabaseUtils.php#L28) to do this.